.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: question
+about: Ask a question regarding the project
+title: "[Question]"
+labels: question
+assignees: adrianoamaral
+
+---
+
+**Ask a question regarding RediSearch**
+
+
+**Additional context**
+Add any other context or screenshots about the question here.
```

Comment:
@DvirDukhan Why not try asking up front for more specific information such as the version, platform, use case, or other details that would allow us to reply with less _ping-pongs_